api/src/functions/health-check/function.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
```

Comment:
Let's change "authLevel" to "anonymous". I think we are going to be using this for nearly all of our methods, since we are doing auth within the function (where applicable, no auth at all in the case of a function like this).

Reply:
This is complete. Definitely makes sense. Thanks!